docs/CHANGELOG.md

```diff
@@ -48,6 +48,7 @@ The sandbox cluster installation is running under the constant load generated by
 * FEATURE: [vmauth](https://docs.victoriametrics.com/vmauth.html): do not print logs like `SIGHUP received...` once per configured `-configCheckInterval` cmd-line flag. This log will be printed only if config reload was invoked manually. 
 * FEATURE: [vmalert](https://docs.victoriametrics.com/vmalert.html): add `eval_offset` attribute for [Groups](https://docs.victoriametrics.com/vmalert.html#groups). If specified, Group will be evaluated at the exact time offset on the range of [0...evaluationInterval]. The setting might be useful for cron-like rules which must be evaluated at specific moments of time. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/3409) for details.
 * FEATURE: [vmalert](https://docs.victoriametrics.com/vmalert.html): validate [MetricsQL](https://docs.victoriametrics.com/MetricsQL.html) function names in alerting and recording rules when `vmalert` runs with `-dryRun` command-line flag. Previously it was allowed to use unknown (aka invalid) MetricsQL function names there. For example, `foo()` was counted as a valid query. See [this feature request](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/4933).
+* FEATURE: [vmalert](https://docs.victoriametrics.com/vmalert.html): add `query_time_alignment` attribute for [Groups](https://docs.victoriametrics.com/vmalert.html#groups), it will align group evaluation time with interval like `datasource.queryTimeAlignment` does. And `datasource.queryTimeAlignment` will be disable by default and deprecated soon. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/5049).
```

Comment:
I think we should add a special deprecation notice to the top of this `tip` section mentioning the breaking change. As we did for some other releases when introducing similar changes.

Reply:
Updated, plz check)